tsconfig.json

```diff
@@ -0,0 +1,19 @@
+{
+  "compilerOptions": {
```

Comment:
I guess we should you same tsconfig for all project like https://github.com/WiseEngineering/crispy-cli/blob/master/tsconfig.json and https://github.com/WiseEngineering/crispy-api/blob/master/tsconfig.json

Reply:
it'll be the different one, we have react here
This one is generated by create-react-app